docs/options api.md

```diff
@@ -129,6 +129,21 @@ class Difficulty(Choice):
     default = 1
 ```
 
+### Option Visibility
+Every option has a Visibility IntFlag, defaulting to `all` (`0b1111`). This lets you choose where the option will be displayed. The flags are:
```

Comment:
Do you think this should mention that the option can still be *used* in all contexts in which you are able to set it? (Which I guess is just yaml)

Reply:
Added something along these lines